xml_inscriptions/transcribed/USEP Penn Museum Amphora Handle.xml

```diff
@@ -0,0 +1,173 @@
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "modern" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date evidence="#archaeology" notBefore="0001" notAfter="0299">first
+                                century to third century CE</date>
+                            <placeName ref="europe.italy.rome">Monte Testaccio</placeName>
+                        </origin>
+                        <provenance type="found">
+                            <p>in <placeName>Monte Testaccio</placeName> in <placeName>Rome</placeName></p>
+                        </provenance>
+                        <provenance type="transferred" subtype="purchased">
+                            <p>acquired by the <orgName>Penn Museum</orgName> through <persName role="funder">John Wanamaker</persName>
+                                in <date when="1896">1896</date></p>
+                        </provenance>
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ref type="external" target="https://www.penn.museum/collections/object/253767">
+                         museum external reference</ref>
+                     
+                    </bibl>
+                </listBibl>
+            </sourceDesc>
+        </fileDesc>
+
+        <!--  *************************************  <encodingDesc>  ********************************* -->
+
+        <encodingDesc>
+
+                <xi:include
+                    href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+                    <xi:fallback>
+                        <p>Taxonomies for US Epigraphy controlled values</p>
+                    </xi:fallback>
+                </xi:include>
+
+        </encodingDesc>
+
+        <profileDesc> <!-- not used --> </profileDesc>
+
+
+        <!--  *************************************  <revisionDesc>  ********************************* -->
+    	<!-- Record all changes here -->
+
+        <revisionDesc>
+            <change when="2021-05-17" who="Colin Olson">Initial Encoding</change>
+        </revisionDesc>
+
+    </teiHeader>
+
+    <!--  *************************************  <facsimile>  *********************************  -->
+        <!-- Used to record images of an inscription.   -->
+
+    <facsimile>
+        <surface>
+            <desc/>
+            <graphic url=""/>
```

Comment:
For reference, if we have an image, its filename should go here in the url; it would likely just match the filename of the xml document (with the appropriate filetype, normally .jpg). If we _don't_ but there happens to be one on a museum website, e.g., you can put a direct link here instead.